lms/models/organization.py

```diff
@@ -3,20 +3,19 @@
 from sqlalchemy.orm import Mapped, mapped_column
 
 from lms.db import Base
-from lms.models._mixins import CreatedUpdatedMixin, PublicIdMixin
+from lms.models._mixins import CreatedUpdatedMixin
 from lms.models.json_settings import JSONSettings
 
 
-class Organization(CreatedUpdatedMixin, PublicIdMixin, Base):
+class Organization(CreatedUpdatedMixin, Base):
     """Model for Organizations comprised of application instances."""
 
     __tablename__ = "organization"
 
-    public_id_model_code = "org"
-    """The short code which identifies this type of model in PublicIdMixin."""
-
     id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
 
+    public_id: Mapped[str] = mapped_column(sa.TEXT(), unique=True)
```

Comment:
So `Organization.public_id` is just a string now :+1:

Is the `sa.TEXT()` here necessary? Doesn't SQLAlchemy infer that from the `Mapped[str]`? (Also it looks like it was `UnicodeText` in the old `PublicIdMixin`.)

Reply:
Just the str would generate a migration, added the type to avoid that.